ooniapi/services/oonimeasurements/src/oonimeasurements/routers/data/utils.py

```diff
@@ -50,3 +52,11 @@ def test_name_to_group(tn):
         return "websites"
     # TODO(arturo): currently we only support websites
     return ""
+
+
+def parse_probe_asn(probe_asn):
+    if probe_asn.startswith("AS"):
```

Comment:
I don't think this is correct. If `probe_asn` is None or `int`, which is supported by the spec, it will raise an exception in here.

Reply:
Actually this is called after checking that it is not None always. About the int, I was testing and always I detected that really is a int as a string, this is the reason about the next lines. But maybe I don't find it the flow with int. Could you show this flow?